app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def show_post
+    @post = Post.find(params[:id])
+  end
```

Comment:
- [ ] Great job following the previous reviewer's advice! 💯  However, please avoid changing the name of the routes, by convention, the restful routes are #index, #show, #new, #create, #edit, #update, #delete. Rails can handle these "Automagically" but when you use custom names, you're the one responsible for connecting everything together: Controller and views. Please consider changing the name of your action to "show". 🤓 🔧